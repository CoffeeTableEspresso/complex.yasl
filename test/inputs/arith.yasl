const complex = __require_c__('complex')

const k = complex(1, 2)
const n = complex(3, -1)

echo k
echo n

echo -n
echo +n

echo n * k
echo n // k
echo n % k

echo n + k
echo n - k
