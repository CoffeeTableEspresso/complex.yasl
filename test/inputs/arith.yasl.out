1.100000 + 2.500000i
3.900000 + -1.000000i
6.790000 + 8.650000i
0.239946 + -1.454424i
5.000000 + 1.500000i
2.800000 + -3.500000i
